### ISSUE TITLE
Add Date to EmailComplexType

### ISSUE THREAD

**Issue by burnalting** [gchq/event-logging-schema#74]
Not all emails have a parseable Date timestamp, but it is the timestamp recorded of the email. We should have a Date String element within the EmailComplextType to represent accurately the Date timestamp recorded in an email.

**Comment by burnalting** [comment]
```
<xs:element name="Date" type="xs:string" minOccurs="0" maxOccurs="1">
  <xs:annotation>
    <xs:documentation>The Date timestamp recorded in the email.</xs:documentation>
  </xs:annotation>
</xs:element>
```

**Comment by at055612** [comment]
Fixed in 4.1.0
